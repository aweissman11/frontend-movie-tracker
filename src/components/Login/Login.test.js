import { Login } from './index';

describe('Login', () => {
  // React component tests
  it('should pass', () => {
    // console.log('pass')
  })
})