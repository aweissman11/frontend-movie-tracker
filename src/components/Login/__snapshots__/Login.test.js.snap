// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Login />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Movie Tracker
        </h1>,
        <input
          className="name-input"
          name="name"
          onChange={[Function]}
          placeholder="What is your Name?"
          value=""
        />,
        <input
          className="email-input"
          name="email"
          onChange={[Function]}
          placeholder="email"
          value=""
        />,
        <input
          className="password-input"
          name="password"
          onChange={[Function]}
          placeholder="password"
          type="password"
          value=""
        />,
        <input
          className="password-confirm-input"
          name="confirmPassword"
          onChange={[Function]}
          placeholder="confirm password"
          type="password"
          value=""
        />,
        <button
          className="create-user-btn"
          onClick={[Function]}
        >
          Submit New User
        </button>,
        <input
          className="login-submit"
          type="submit"
          value="Login"
        />,
        <button>
          Sign Up
        </button>,
        <Link
          replace={false}
          to="/release-date"
        >
          <button>
            Skip login
          </button>
        </Link>,
        <Link
          replace={false}
          to="/login"
        >
          <button>
            Already a user
          </button>
        </Link>,
        <div
          className="login "
        />,
        <div
          className="sign up "
        />,
      ],
      "className": "login-page",
      "onSubmit": [Function],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Movie Tracker",
        },
        "ref": null,
        "rendered": "Movie Tracker",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "name-input",
          "name": "name",
          "onChange": [Function],
          "placeholder": "What is your Name?",
          "value": "",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "email-input",
          "name": "email",
          "onChange": [Function],
          "placeholder": "email",
          "value": "",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "password-input",
          "name": "password",
          "onChange": [Function],
          "placeholder": "password",
          "type": "password",
          "value": "",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "password-confirm-input",
          "name": "confirmPassword",
          "onChange": [Function],
          "placeholder": "confirm password",
          "type": "password",
          "value": "",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Submit New User",
          "className": "create-user-btn",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": "Submit New User",
        "type": "button",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "login-submit",
          "type": "submit",
          "value": "Login",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Sign Up",
        },
        "ref": null,
        "rendered": "Sign Up",
        "type": "button",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <button>
            Skip login
          </button>,
          "replace": false,
          "to": "/release-date",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Skip login",
          },
          "ref": null,
          "rendered": "Skip login",
          "type": "button",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <button>
            Already a user
          </button>,
          "replace": false,
          "to": "/login",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Already a user",
          },
          "ref": null,
          "rendered": "Already a user",
          "type": "button",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "login ",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "sign up ",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
    ],
    "type": "form",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Movie Tracker
          </h1>,
          <input
            className="name-input"
            name="name"
            onChange={[Function]}
            placeholder="What is your Name?"
            value=""
          />,
          <input
            className="email-input"
            name="email"
            onChange={[Function]}
            placeholder="email"
            value=""
          />,
          <input
            className="password-input"
            name="password"
            onChange={[Function]}
            placeholder="password"
            type="password"
            value=""
          />,
          <input
            className="password-confirm-input"
            name="confirmPassword"
            onChange={[Function]}
            placeholder="confirm password"
            type="password"
            value=""
          />,
          <button
            className="create-user-btn"
            onClick={[Function]}
          >
            Submit New User
          </button>,
          <input
            className="login-submit"
            type="submit"
            value="Login"
          />,
          <button>
            Sign Up
          </button>,
          <Link
            replace={false}
            to="/release-date"
          >
            <button>
              Skip login
            </button>
          </Link>,
          <Link
            replace={false}
            to="/login"
          >
            <button>
              Already a user
            </button>
          </Link>,
          <div
            className="login "
          />,
          <div
            className="sign up "
          />,
        ],
        "className": "login-page",
        "onSubmit": [Function],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Movie Tracker",
          },
          "ref": null,
          "rendered": "Movie Tracker",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "name-input",
            "name": "name",
            "onChange": [Function],
            "placeholder": "What is your Name?",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "email-input",
            "name": "email",
            "onChange": [Function],
            "placeholder": "email",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "password-input",
            "name": "password",
            "onChange": [Function],
            "placeholder": "password",
            "type": "password",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "password-confirm-input",
            "name": "confirmPassword",
            "onChange": [Function],
            "placeholder": "confirm password",
            "type": "password",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Submit New User",
            "className": "create-user-btn",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": "Submit New User",
          "type": "button",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "login-submit",
            "type": "submit",
            "value": "Login",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sign Up",
          },
          "ref": null,
          "rendered": "Sign Up",
          "type": "button",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <button>
              Skip login
            </button>,
            "replace": false,
            "to": "/release-date",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Skip login",
            },
            "ref": null,
            "rendered": "Skip login",
            "type": "button",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <button>
              Already a user
            </button>,
            "replace": false,
            "to": "/login",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Already a user",
            },
            "ref": null,
            "rendered": "Already a user",
            "type": "button",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "login ",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "sign up ",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
      ],
      "type": "form",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
